apps/docs/src/app/platform/component-docs/platform-action-bar/platform-action-bar-header/platform-action-bar-header.component.html

```diff
@@ -1,8 +1,35 @@
 <header>Action Bar</header>
 <description>
-    The Action Bar is located at the top of the page and is used for Page title and Main Actions for the page.
+    <p>
+        Platform ActionBar component is built on top of
+        <a [routerLink]="'/core/action-bar'">fundamental-ngx/core</a>
+        ActionBar component to provide additional enhancements.

```

Comment:
I don't think this sentence is necessary.

Reply:
removed sentence.